samples/attested_tls/README.md

```diff
@@ -93,11 +93,11 @@ Note: Both of them can run on the same machine or separate machines.
   make run
   ```
 
-- Use FIPS-enabled OpenSSL based on SymCrypt
+- Use FIPS-enabled OpenSSL based on SymCrypt engine
   ```bash
   mkdir build
   cd build
-  cmake -DOE_CRYPTO_LIB=openssl_symcrypt_fips ..
+  cmake -DOE_CRYPTO_LIB=symcrypt_fips ..
```

Comment:
I understand why you are trying to make this change, but I would like to keep this as is. Can you please keep this as is and add openssl3_symcrypt_fips as the new option?

Reply:
This change makes it in line with the naming system used in the main codebase. It's also a more accurate description of what's going on because technically you can use the symcrypt engine with openssl 3 if deprecated support is enabled.